.github/workflows/update_knapsack_report.yml

```diff
@@ -29,14 +29,14 @@ jobs:
 
     env:
       RAILS_ENV: test
+      github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
where is this ENV VAR used? (not the secrets)

Reply:
Thanks. I'll remove